src/pages/Login/Login.js

```diff
@@ -1,18 +1,72 @@
-import React from "react";
+import React, { useState } from "react";
+import axios from "axios";
+import { useAuth } from "../../Context/Auth-Context";
+import { Link, useNavigate } from "react-router-dom";
 import "./login.css";
-import { Link } from "react-router-dom";
 const Login = () => {
+  const { authDispatch } = useAuth();
+  const navigateProduct = useNavigate();
+  const [loginData, setLoginData] = useState({ email: "", password: "" });
+  const getLoginData = (e) => {
+    const { name, value } = e.target;
+    setLoginData({ ...loginData, [name]: value });
+  };
+  const testUserCredential = {
+    email: "joeytribbiani@gmail.com",
+    password: "joeytribbiani",
+  };
+  const testCredential = (e) => {
+    e.preventDefault();
+    setLoginData(testUserCredential);
+  };
+
+  const loginHandler = async (e) => {
+    e.preventDefault();
+    try {
+      const res = await axios.post("/api/auth/login", {
+        email: loginData.email,
+        password: loginData.password,
+      });
+      if (res.status === 200) {
+        localStorage.setItem("token", res.data.encodedToken);
```

Comment:
I would suggest instead of giving key name as only `token`, go with `projectName.token`. so it won't conflict with other projects during development